Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -94,8 +97,37 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 xASL_delete(rpfile);
 
 % Run SPM
-if nFrames>2 && bSubtraction
+
+%####### Calculating the motion for every first TE of each PLD and
+% considering it the same for the other TEs from that PLD.
+%
+% if nFrames>2 && bSubtraction && length(x.EchoTime)>1  %Multi TE 
+%     uniqueTE=uniquetol(x.EchoTime); %gives the number of unique TEs

```

Comment:
@BeatrizPadrela Should we really use `uniquetol` here or would `unique` be better? We haven't specified the tolerance here...